core/app-framework/app-native-shared/bi-inc/attr_container.h

```diff
@@ -20,13 +20,17 @@ extern "C" {
 
 /* Attribute type */
 enum {
-    ATTR_TYPE_BEGIN = 1,
-    ATTR_TYPE_SHORT = ATTR_TYPE_BEGIN,
+    ATTR_TYPE_BEGIN = 0,
+    ATTR_TYPE_BYTE = ATTR_TYPE_BEGIN,
+    ATTR_TYPE_INT8 = ATTR_TYPE_BYTE,
+    ATTR_TYPE_SHORT,
     ATTR_TYPE_INT,
```

Comment:
Had better change them to `ATTR_TYPE_INT16` and `ATTR_TYPE_INT32`, or keep them and add the later:
```C
ATTR_TYPE_INT16,
ATTR_TYPE_SHORT = ATTR_TYPE_INT16,
ATTR_TYPE_INT32,
ATTR_TYPE_INT = ATTR_TYPE_INT32
```
And for the API `attr_container_{set|get}_{int|short}`, had better change them to `attr_container_{set|get}_{int32|int16}`, or keep them, and add `attr_container_{set|get}_{int32|int16}`?

Reply:
Ok, I got it. I will change it.